music21/abcFormat/__init__.py

```diff
@@ -1766,11 +1783,18 @@ def barlineTokenFilter(token):
         else: # append unaltered
             post.append(ABCBar(token))
         return post
-
-
+    
     # --------------------------------------------------------------------------
     # token processing
 
+    def _accidentalPropagation(self):
```

Comment:
Can this be tested here with a doctest and explained in a doc string (required even for "private" functions)?  What about %abc-2.0?

Reply:
(I just don't want to fix a weird bug when I'm 60 or because the version numbering system of abc changes and ABC 10.0 is released)

Reply:
Agreed. I have added the future-proofing.

Also, 2.0 is included now. For some reason, I thought this was only ever a draft standard.